cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteServiceManager.java

```diff
@@ -40,10 +42,10 @@
   private final ServiceClient serviceClient;
   private final Id.Service serviceId;
 
-  public RemoteServiceManager(Id.Service programId, ClientConfig clientConfig, RESTClient restClient,
+  public RemoteServiceManager(ProgramId programId, ClientConfig clientConfig, RESTClient restClient,
```

Comment:
But this API is not good now, right? I can even pass a non-service ``ProgramId`` to this constructor. This should only take ``ServiceId`` and not ``ProgramId``.